src/Apps/Artwork/Components/PrivateArtwork/__tests__/PrivateArtworkAboutWork.jest.tsx

```diff
@@ -0,0 +1,30 @@
+import { screen } from "@testing-library/react"
+import { PrivateArtworkAboutWork } from "Apps/Artwork/Components/PrivateArtwork/PrivateArtworkAboutWork"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { graphql } from "react-relay"
+import { PrivateArtworkAboutWorkQuery } from "__generated__/PrivateArtworkAboutWork_artwork.graphql"
+
+jest.unmock("react-relay")
+jest.mock("System/useFeatureFlag", () => {
+  return {
+    useFeatureFlag: jest.fn().mockReturnValue(true),
+  }
+})
+
+describe("PrivateArtworkAboutWork", () => {
+  const { renderWithRelay } = setupTestWrapperTL<PrivateArtworkAboutWork_artwork>({
+    Component: props => <PrivateArtworkAboutWork artwork={props.artwork!} />,
+    query: graphql`
+      query PrivateArtworkAboutWorkQuery {
+        artwork(id: "foo") {
+          ...PrivateArtworkAboutWork_artwork
+        }
+      }
+    `,
+  })
+
+  it("displays artwork information", async () => {
+    renderWithRelay({})
+    expect(await screen.findByText("")).toBeInTheDocument()
```

Comment:
as mentioned above, perhaps getByPlaceholder instead of findByText. also I don't think these are async so I will probably remove the await in that case.